dockerfiles/Dockerfile.tensorrt

```diff
@@ -0,0 +1,57 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with TensorRT integration
+
+# nVidia TensorRT Base Image
+FROM nvcr.io/nvidia/tensorrt:19.03-py3
+
+# Add conda to PATH
+WORKDIR /work/deps
+
+# Dependencies: Ubuntu-core
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends sudo \
+    libopenblas-base \
+    wget \
+    zip \
+    build-essential curl libcurl4-openssl-dev libssl-dev python3-dev git
+
+# Dependencies: conda
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.11-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/miniconda && \
+    rm ~/miniconda.sh && \
+    /opt/miniconda/bin/conda clean -tipsy
+
+RUN conda install -y python=3.6 numpy && \
+    conda clean -aqy && \
+    rm -rf /opt/miniconda/pkgs && \
+    find / -type d -name __pycache__ -prune -exec rm -rf {} \;
+
+# Build the latest cmake
+WORKDIR /code
+RUN wget --quiet https://github.com/Kitware/CMake/releases/download/v3.14.3/cmake-3.14.3-Linux-x86_64.tar.gz
+RUN tar zxf cmake-3.14.3-Linux-x86_64.tar.gz
+
+ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:/opt/miniconda/bin:${PATH}
+
+# Prepare onnxruntime repository
+RUN git clone --recursive https://github.com/Microsoft/onnxruntime -b master --single-branch
+WORKDIR /code/onnxruntime
+
+# Start the TensorRT build
+RUN /bin/sh ./build.sh --cuda_home /usr/local/cuda \
+--cudnn_home /usr/lib/x86_64-linux-gnu/ --use_tensorrt --tensorrt_home /workspace/tensorrt \
```

Comment:
I think --tensorrt_home /workspace/tensorrt was a mistake. (though it might not matter)
the tensorrt libs are actually in /usr/lib/x86_64-linux-gnu/

Reply:
The base nVidia TensorRT container has a workspace/tensorrt link with the libs in it!